src/ucp/rma/amo_sw.c

```diff
@@ -135,12 +135,12 @@ static ucs_status_t ucp_progress_atomic_reply(uct_pending_req_t *self)
 }
 
 #define DEFINE_AMO_SW_OP(_bits) \
-    static void ucp_amo_sw_do_op##_bits(const ucp_atomic_req_hdr_t *atomicreqh) \
+    static void ucp_amo_sw_do_op##_bits(const ucp_atomic_req_hdr_t *amo_req_hdr) \
     { \
-        uint##_bits##_t *ptr  = (void*)atomicreqh->address; \
-        uint##_bits##_t *args = (void*)(atomicreqh + 1); \
+        uint##_bits##_t *ptr  = (void*)amo_req_hdr->address; \
```

Comment:
such rename/cleanup better be in another pr

Reply:
ok